helper/keysutil/policy.go

```diff
@@ -702,8 +714,15 @@ func (p *Policy) Encrypt(ver int, context, nonce []byte, value string) (string,
 	// Convert to base64
 	encoded := base64.StdEncoding.EncodeToString(ciphertext)
 
+	template := p.VersionTemplate
+	if template == "" {
+		template = DefaultVersionTemplate
+	}
+
+	prefix := strings.Replace(template, "{{version}}", strconv.Itoa(ver), -1)
```

Comment:
It'd be nice to cache the result (potentially when the policy is loaded) so that we don't need to take the hit performing string substitution on every encrypt/decrypt.

Reply:
Github messed up on hiding this comment, this is done.